### ISSUE TITLE
Android: Separate traffic tunneling

### ISSUE THREAD

**Issue by birykvlad** [Jigsaw-Code/outline-client#1618]
**What would you like the app to enable you to do?**
I want to implementation of split traffic tunneling.
The ability to select applications that should go through the VPN connection and which should not. Many popular VPN applications have this feature.

**Describe alternatives you've considered**
Red Shield VPN, NordVPN, Proton VPN.

**Comment by master88wi** [comment]
+1

**Comment by AliKarami** [comment]
+1

**Comment by rnabiullin** [comment]
+1

**Comment by Lucius-Will** [comment]
Yes. It's a very important feature. It's sorely lacking. In Russia, VPN is no longer a whim, but a necessity. Hello North Korea 2.0 🥺

**Comment by Lucius-Will** [comment]
TipTop VPN can also do this. But RosComNadzor has been periodically blocking it as well.

**Comment by dazuiba** [comment]
For anyone still looking for split tunneling with Outline

I built a free third-party client called Dash for Outline that works with existing Outline servers and adds rule-based routing on macOS and iOS:

http://getdashvpn.com/  (not affiliated with Outline Foundation)
